### PR TITLE
Minor bug fix to recovery services backup

### PR DESCRIPTION
**Pull request by AdityaTulasi-xx**
Undoing unintended removal of a field in two classes

### CODE REVIEW

File: src/SDKs/RecoveryServices.Backup/Management.RecoveryServices.Backup/Generated/Models/PreValidateEnableBackupRequest.cs [hunk 1]
```diff
@@ -74,6 +77,12 @@ public PreValidateEnableBackupRequest(string resourceType = default(string), str
         [JsonProperty(PropertyName = "resourceId")]
         public string ResourceId { get; set; }
 
+        /// <summary>
```
Comment (dsgouda) [inline src/SDKs/RecoveryServices.Backup/Management.RecoveryServices.Backup/Generated/Models/PreValidateEnableBackupRequest.cs]: Please bump the Nuget minor version (in .csproj and AssemblyInfo.cs) and update PackageReleaseNotes

**Review by dsgouda — CHANGES_REQUESTED** [review]
How was this code generated. Even if it is changes to minor fields, they must be fixed in the swagger spec first and the code has to be regenerated. Please link the swagger spec changes' PR

**Comment by AdityaTulasi** [comment]
>           How was this code generated. Even if it is changes to minor fields, they must be fixed in the swagger spec first and the code has to be regenerated. Please link the swagger spec changes' PR

Link to spec PR - https://github.com/Azure/azure-rest-api-specs/pull/5133

**Review by dsgouda — APPROVED** [review]
LGTM

**Comment by dsgouda** [comment]
WIll merge once CIs pass

**Comment by dsgouda** [comment]
@AdityaTulasi please join the azure org [here](aka.ms/azuregithub) to kick off the build

**Review by dsgouda — CHANGES_REQUESTED** [review]
Please regenerate the code using generate.ps1 script. This should generate changes to SdkInfo_*.cs and .txt files. Please commit these changes too.

**Review by dsgouda — APPROVED** [review]
LGTM, will merge when CIs pass

**Comment by dsgouda** [comment]
@AdityaTulasi could you pull latest changes from master and update this PR

**Pull request merged**